eclipse-theia/theia
# Open external files by DND into main panel

#### What it does
Fixes https://github.com/eclipse-theia/theia/issues/9543
Drag and dropping files from the file system into the main panel now opens their respective editors. Previously, files that could not be matched to a navigator tree node were ignored.

#### How to test
1. Start the electron app
2. Drag a file that does not belong to the workspace into the editor area and drop it
3. The editor for the respective file should open

The same should work if multiple files are selected: an editor for each file should open.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: Alexandra Buzila <abuzila@eclipsesource.com>
